megamek/src/megamek/common/Entity.java

```diff
@@ -14282,8 +14280,7 @@ private boolean doMASCOrSuperchargerFailureCheckFor(Mounted masc, Vector<Report>
             r.add(getMASCTarget());
             r.add(nRoll);
 
-            if (
-                    (!isSupercharger && nRoll < getMASCTarget())
+            if ( (!isSupercharger && nRoll < getMASCTarget())
```

Comment:
```suggestion
            if ((!isSupercharger && (nRoll < getMASCTarget()))
```
The next line is also missing internal brackets around the second comparison.